scripts/publish.js

```diff
@@ -90,7 +90,7 @@ shell.exec('git clean -xfd -e node_modules');
 shell.exec('npm install');
 
 // Compile
-shell.exec('npm run compile');
+shell.exec('NODE_ENV=production npm run compile');
```

Comment:
This allows us to run webpack in production mode when publishing the extensions.